doc/source/docs/user_guide/mapping.rst

```diff
@@ -63,37 +63,39 @@ When plotting a map, one can enable a legend using the ``legend`` argument:
 .. ipython:: python
 
     # Plot population estimates with an accurate legend
-    import matplotlib.pyplot as plt
-    fig, ax = plt.subplots(1, 1)
-    @savefig world_pop_est.png
-    world.plot(column='pop_est', ax=ax, legend=True)
+    @savefig chicago_choro.png
+    chicago.plot(column='Pop2014', legend=True);
 
-However, the default appearance of the legend and plot axes may not be desirable. One can define the plot axes (with ``ax``) and the legend axes (with ``cax``) and then pass those in to the :meth:`~GeoDataFrame.plot` call. The following example uses ``mpl_toolkits`` to vertically align the plot axes and the legend axes:
+The following example plots the color bar below the map and adds its label using ``legend_kwds``:
 
 .. ipython:: python
 
     # Plot population estimates with an accurate legend
-    from mpl_toolkits.axes_grid1 import make_axes_locatable
-    fig, ax = plt.subplots(1, 1)
-    divider = make_axes_locatable(ax)
-    cax = divider.append_axes("right", size="5%", pad=0.1)
-    @savefig world_pop_est_fixed_legend_height.png
-    world.plot(column='pop_est', ax=ax, legend=True, cax=cax)
-
+    @savefig chicago_horizontal.png
+    chicago.plot(
+        column="Pop2014",
+        legend=True,
+        legend_kwds={"label": "Population in 2014", "orientation": "horizontal"},
```

Comment:
Similarly, the actual map is now very tiny, while the colorbar is quite large when laid out horizontally.

Reply:
Made the map a bit larger. Don't want to go much further to avoid too-large figures.